modules/accessing-hosts-on-aws.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * networking/accessing-hosts.adoc
+
+[id="accessing-hosts-on-aws_{context}"]
+= Accessing hosts on Amazon Web Services (AWS) after a full-stack automation (IPI) installation
+
+The {product-title} installer does not create any public IP addresses for any of
+the Amazon Elastic Compute Cloud (Amazon EC2) instances that it provisions for
+your {product-title} cluster. In order to be able to SSH to your {product-title}
+hosts, you must follow this procedure.
+
+.Procedure
+
+. Create a security group that allows SSH access into the virtual private cloud
+(VPC) created by the `openshift-install` command.
+
+. Create an Amazon EC2 instance inside the VPC that the installer created or on
```

Comment:
If want to user can ssh login bastion host, I think `Create an Amazon EC2 instance on one of the public subnets the installer created` will be more accurate, no need other option, such as: `inside the VPC that the installer created`.

Reply:
Thank you!